tests/test_watson.py

```diff
@@ -8,9 +8,9 @@
     import mock
 
 try:
-    from io import StringIO
-except ImportError:
     from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
:+1:, `io.StringIO` doesn't have the right behavior in Python 2.7. This is very confusing.